ssl/statem/statem_quic.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
This copyright seems different from the other ones that say 2018-2021. Why?

Reply:
It's when the code was originally written! :D The copyright can be updated in a clean up commit.